src/modules/test/commentsCount.test.js

```diff
@@ -0,0 +1,22 @@
+import commentCounter from '../commentscount.js';
+
+global.fetch = jest.fn();
+describe('commentCounter', () => {
+  beforeEach(() => {
+    fetch.mockReset();
+  });
+
+  test('returns the number of comments', async () => {
+    const comment = [{ id: 1 }, { id: 2 }, { id: 3 }];
+    fetch.mockResolvedValue({ json: () => Promise.resolve(comment) });
+    const result = await commentCounter();
+    expect(result.length).toBe(3);
```

Comment:
- [ ] As suggested by the previous reviewer, please kindly add all edge test cases. E.G
- the scenario in which there 0 items on page or 0 comments

Reply:
Thank you @topeogunleye we are fixing it

Reply:
Great 👍 , Keep up the great work

Reply:
@ishimwezachee @topeogunleye Hey guys, if any of you is available, can you please check out our PR, we can’t seem to get anyone for the review.